app/_how-tos/protect-sensitive-information-output-with-ai.md

```diff
@@ -0,0 +1,191 @@
+---
+title: Use AI PII Sanitizer plugin to protect sensitive data in responses
+content_type: how_to
+
+description: Use the AI PII Sanitizer plugin to protect sensitive information in responses from a Mistral LLM model.
+
+entities:
+  - certificate
+  - service
+
+products:
+    - gateway
+    - ai-gateway
+
+works_on:
+    - on-prem
+    - konnect
+
+tools:
+    - deck
+
+plugins:
+  - ai-proxy
+  - ai-sanitizer
+  - file-log
+
+tags:
+  - ai
+  - security
+
+tldr:
+  q: How can I anonymize sensitive information in API responses using AI?
+  a: Enable the [AI Proxy](/plugins/ai-proxy/) and then [AI PII Sanitizer](/plugins/ai-sanitizer) plugin in `OUTPUT` mode to automatically redact or replace sensitive data in the responses from your service. Then, use [File Log](/plugins/file-log) plugin to audit what PII data was sanitized
+
+prereqs:
+  entities:
+    services:
+      - example-service
+    routes:
+      - example-route
+  inline:
+    - title: Mistral
+      include_content: prereqs/mistral
+      icon_url: /assets/icons/mistral.svg
+    - title: AI PII Anonymizer service access
+      include_content: prereqs/ai-sanitizer
+      icon_url: /assets/icons/cloudsmith.svg
+
+cleanup:
+  inline:
+    - title: Clean up Konnect environment
+      include_content: cleanup/platform/konnect
+      icon_url: /assets/icons/gateway.svg
+    - title: Destroy the {{site.base_gateway}} container
+      include_content: cleanup/products/gateway
+      icon_url: /assets/icons/ai.svg
+
+min_version:
+  gateway: '3.12'
+
+related_resources:
+  - text: Use AI PII Sanitizer plugin to protect sensitive information in responses
+    url: /how-to/protect-sensitive-information-output-with-ai/
+  - text: AI PII Sanitizer
+    url: /plugins/ai-sanitizer/
+
+automated_tests: false
+---
+## Start the Kong AI PII Sanitizer service
+
+Make sure you have [access to the  AI PII service](#ai-pii-anonymizer-service-access), then run the following command to start it locally with Docker:
+
+```sh
+docker run --rm -p 8080:8080 docker.cloudsmith.io/kong/ai-pii/service:v0.1.2-en
+```
+
+## Enable the AI Proxy plugin
+
+Use the AI Proxy plugin to connect to Mistral provider:
```

Comment:
"To connect to Mistral" or "to connect to the Mistral provider" - one of those two options would be better. This one is halfway.